contracts/message-bus/MessageBus.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "./MessageBox.sol";
+import "./Proof.sol";
+import "./MessageInbox.sol";
+import "./MessageOutbox.sol";
+
+contract MessageBus is MessageOutbox, MessageInbox {
```

Comment:
Not sure this abstraction is very valuable. Child contract can choose to extends `either` or both contracts (MessageOutbox and MessageInbox).

Reply:
A child contract still can choose it.